### DISCUSSION TITLE
App running on a locked device (Android)

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#3614] [General]
How can I make my flet application remain running even when the device is locked?

Situation where this would be necessary:
Imagine that I developed a running app, so I need my app to run even when the device is locked.

Is this already possible with flet?

**Comment by burhansvural** [comment]
add in manifest file => android:showWhenLocked="true"

**Reply by nascin** [reply]
Insert the manifest file in the root of my Flat project?

**Reply by burhansvural** [reply]
Manifest file in templates @ndonkoHenri and @FeodorFitsner know better how to use templates.

**Reply by nascin** [reply]
@FeodorFitsner, is there a programmatic way to do this, or has it not been implemented yet?

**Reply by ndonkoHenri** [reply]
Hasn't yet.
